src/utils/time.ts

```diff
@@ -196,7 +196,7 @@ function getSunsetSunriseUTCTime(
         return {
             alwaysDay: false,
             alwaysNight: false,
-            time: UT * getDuration({hours: 1}),
+            time: Math.round(UT * getDuration({hours: 1})),
```

Comment:
This rounds number of milliseconds to the nearest integer, which is convenient for debugging and writing tests. This rounding does not actually lead to any perceived precision loss since JS event loop can introduce delays over a millisecond anyway.